src/Organizations/OrganizationDetails/OrganizationContactPeople/OrganizationContactPeople.js

```diff
@@ -26,7 +26,7 @@ const alignRowProps = { alignLastColToEnd: true };
 
 const OrganizationContactPeople = ({ vendorCategories, contacts, openContact }) => {
   const resultsFormatter = {
-    name: ({ firstName, lastName }) => `${firstName} ${lastName}`,
+    name: ({ firstName, lastName }) => `${lastName}, ${firstName}`,
```

Comment:
IMHO, you might want to use common code `import { getFullName } from '@folio/stripes/util';`

Reply:
It works for user schema only
https://github.com/folio-org/stripes-util/blob/master/lib/getFullName.js#L7